README.md

```diff
@@ -1,2 +1,28 @@
-# emmanuel_tic-tac-toe
-A simple command line game of Tic Tac Toe in Ruby
+# TicTacToe
+
+[![Build Status](https://travis-ci.org/oneworldcoders/emmanuel_tic-tac-toe.svg?branch=master)](https://travis-ci.org/oneworldcoders/emmanuel_tic-tac-toe)
+
+A simeple command line application of an implementation of tic tac toe in ruby following the TDD approach
+
+
+## Installation
+
+After cloning the repo
+
+Run:
+
+    $ bundle
+
+Or install it yourself as:
+
+    $ gem install tic_tac_toe
+
+## Usage
+
+TODO: Write usage instructions here
+
+## Development
```

Comment:
I think the section below is more pertinent to the gem setup. For this project, perhaps you could include only instruction on how to run test `rspec spec` or `rake spec` depending on the setup.